dwave_networkx/generators/chimera.py

```diff
@@ -65,6 +73,18 @@ def chimera_graph(m, n=None, t=None, create_using=None, node_list=None, edge_lis
         If True, node labels are 4-tuples, equivalent to the chimera_index
         attribute as below.  In this case, the `data` parameter controls the
         existence of a `linear_index attribute`, which is an int.
+    check_node_list : bool (optional, default False)
+        If True, the node_list elements are checked for compatibility with
+        the graph topology and node labeling conventions, an error is thrown
+        if any node is incompatible or duplicates exist. 
+        In other words, only node_lists that specify subgraphs of the default 
```

Comment:
"only node lists that specify"

By "the default" do you mean the one for the specified or default ``m, n, t`` Chimera?

Reply:
I'm using by default several times in these document strings to mean when all parameters are defaulted, there is a lot of interplay between parameters unfortunately, particularly the pegasus case. I can just avoid 'by default' and be more specific.